packages/vapor/scss/typography/headings.scss

```diff
@@ -66,3 +60,21 @@ h6 {
 .mod-header-min-height {
     min-height: $header-line-height;
 }
+
+.mod-h1-margin {
+    margin-top: 1.2em;
+}
+
+.mod-h2-margin {
+    margin-top: 1.3em;
+}
+
+.mod-h3-margin {
+    margin-top: 1.4em;
+}
+
+.mod-h4-margin,
+.mod-h5-margin,
+.mod-h6-margin {
+    margin-top: 1.5em;
+}
```

Comment:
I don't know if we can do something like:
```scss
.mod-margin {
  &.h1, h1& { margin-top: 1.2em}
  &.h2, h2& { margin-top: 1.3em}
  &.h3, h3& { margin-top: 1.4em}
  &.h4, h4&, &.h5, h5&, &.h6, h6& { margin-top: 1.5em}
}
```

Which would be used like `<h1 class="mod-margin">Title</h1>` or `<span class="h3 mod-margin">Title</span>`